central/notifiers/syslog/syslog.go

```diff
@@ -149,10 +160,10 @@ func joinRoleNames(roles []*storage.UserInfo_Role) string {
 	return strings.Join(roleNames, ",")
 }
 
-func alertToCEF(alert *storage.Alert) string {
+func alertToCEF(alert *storage.Alert, notifier *storage.Notifier) string {
 	// There will be 4-5 different key/value pairs in this message.  Allocate space for 5 because additional
 	// allocations are more expensive than a slightly large list
-	extensionList := make([]string, 0, 5)
+	extensionList := make([]string, 0, 6)
```

Comment:
nit: We need to update the comment above; it still says we are allocating 5 fields, to allow for the maximum list size.